### PR TITLE
Ogre no absolute

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

File: ports/ogre/portfile.cmake [hunk 1]
```diff
@@ -124,7 +124,13 @@ if(VCPKG_TARGET_IS_WINDOWS)
     endforeach()
 endif()
 
+file(GLOB share_cfgs ${CURRENT_PACKAGES_DIR}/share/OGRE/*.cfg)
```
Comment (strega-nil-ms) [inline ports/ogre/portfile.cmake:127]: I also can't test this change; what are these *.cfg files used for?

Comment (autoantwort) [reply ports/ogre/portfile.cmake:127]: e.g.: resources.cfg:
<details>

```
# Ogre Core Resources
[OgreInternal]
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/ShadowVolume
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/RTShaderLib/materials
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/RTShaderLib/GLSL
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/RTShaderLib/HLSL_Cg
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/Terrain/

# Resources required by the sample browser and most samples.
[Essential]
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/SdkTrays.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/profiler.zip
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/thumbnails

# Common sample resources needed by many of the samples.
# Rarely used resources should be separately loaded by the
# samples which require them.
[General]
# Bites uses the next entry to discover the platform shaders
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/fonts
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/materials/programs
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/materials/scripts
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/materials/textures
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/materials/textures/terrain
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/models
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/particle
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/DeferredShadingMedia
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/DeferredShadingMedia/DeferredShading/post
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/PCZAppMedia
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/materials/scripts/SSAO
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/materials/textures/SSAO
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/volumeTerrain
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/CSMShadows
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/cubemap.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/cubemapsJS.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/dragon.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/fresneldemo.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/ogretestmap.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/ogredance.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/Sinbad.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/skybox.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/volumeTerrain/volumeTerrainBig.zip

FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/PBR
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/PBR/filament
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/materials/textures/PBR
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/DamagedHelmet.zip
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/filament_shaders.zip

FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/HLMS

[BSPWorld]
Zip=/home/leander/vcpkg/packages/ogre_x64-linux/share/OGRE/Media/packs/oa_rpg3dm2.pk3

# Materials for visual tests
[Tests]
FileSystem=/home/leander/vcpkg/packages/ogre_x64-linux/Tests/Media
```

</details>
This file is currently only loaded in test. But maybe also by end users and defines where to find stuff

**Comment by Cheney-W** [comment]
@strega-nil-ms  Could you please take a look again? Thank you!

File: ports/ogre/portfile.cmake (hunk 1 above)
Comment (ras0219-msft) [reply ports/ogre/portfile.cmake:127]: According to https://ogrecave.github.io/ogre/api/1.12/setup.html#autotoc_md294:

> Ogre must find plugins.cfg and resources.cfg to function properly.

> [for plugins.cfg] Relative paths are resolved relative to the location of plugins.cfg.
> [for resources.cfg] Relative paths are resolved relative to the location of resources.cfg

Therefore this should replace `${CURRENT_PACKAGES_DIR}` with `../../`

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

**Comment by ras0219-msft** [comment]
LGTM, thanks!

**Pull request merged**